.pre-commit-config.yaml

```diff
@@ -51,15 +54,11 @@ repos:
     # run this hook on `labeler.yml` (or certain other files).
     exclude: .github/labeler.yml|.pre-commit-search-and-replace.yaml
 
-- repo: https://github.com/pre-commit/mirrors-csslint
-  rev: v1.0.5
-  hooks:
-  - id: csslint
```

Comment:
It is quite rare for us to modify a CSS file, so I'm dropping this.  But, we should remember that `csslint` exists for when we do need it one of these years.

Reply:
I'm assuming CSS here is not cascading style sheets?

Reply:
It is indeed cascading style sheets.  We have a couple of CSS files for our website, for better or for worse.

Reply:
Argh I see, I have some previous experience working with web development and hosting. If you need something looking at ever I could probably lend a hand